.gitignore

```diff
@@ -31,4 +31,6 @@
 *.out
 *.app
 
-build
\ No newline at end of file
+build
+
+.vscode
```

Comment:
I think we want this code in the actual repo, can you make a new branch and PR against John's repo to add just the `.vscode`?